packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -42,17 +44,21 @@ class MobileAdTargetingInfo {
   const MobileAdTargetingInfo(
       {this.keywords,
       this.contentUrl,
-      this.birthday,
-      this.gender,
-      this.designedForFamilies,
+      @deprecated this.birthday,
```

Comment:
Any information about how devs should deal with these deprecations?

Reply:
Gender and birthday have been deprecated without replacement upstream.  DesignedForFamilies is now covered by childDirected.

Reply:
Would it be possible to add a comment here then that indicates what devs should expect with these fields? Anything that lets a dev know what to use instead when they see this as being deprecated, in this case nothing.

Reply:
How about like this?

Reply:
That works for me, just fix the formatting and we should be good to go.

Reply:
Should be good now?